puppetserver/Dockerfile

```diff
@@ -110,6 +110,7 @@ RUN apt-get update && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* && \
     gem install --no-doc r10k && \
```

Comment:
should we add a version here to ensure we don't pull in breaking changes by accident?

Reply:
okay, will add...